### DISCUSSION TITLE
How to pass/define JVM options (in keycloak.conf) other than in JAVA_OPTS_APPEND?

### DISCUSSION THREAD

**Discussion by initshdb** [keycloak/keycloak#13495] [Q&A]
In one of our requirement, we are using Oracle Wallet to manage database credentials to Keycloak. Below is the configuration in `keycloak.conf` to enable oracle wallet usage. We are passing the oracle wallet location required as a JVM arg. Can such JVM args be defined in the `keycloak.conf` or `quarkus.properties`, instead of passing explicilty in the JVM options?
```
db=oracle
db-url=jdbc:oracle:thin:@<TNS_ALIAS>?TNS_ADMIN=<tnsnames_ora_location>
```
Additonal JVM Option: `-Doracle.net.wallet_location=<oracle_wallet_location>`

**Comment by ajaiswal595** [comment]
@nitishdb did you get the solution, how to pass jvm option in Quarkus keycloak?
